src/main/java/com/philemonworks/selfdiagnose/SelfDiagnoseServlet.java

```diff
@@ -146,6 +146,10 @@ public void run(HttpServletRequest req, HttpServletResponse resp) throws Servlet
             // bring request available to thread
             SelfDiagnoseServlet.setCurrentRequest(req);
             ExecutionContext ctx = new ExecutionContext();
+            String parallel = req.getParameter("parallel");
+            if (parallel != null) {
+                ctx.setValue("parallel", true);

```

Comment:
here, parallel is now reserved word. This may cause problems with users that already have this taken this name. Better to rename it to something like  "selfdiagnose-tasks-parallel"

Reply:
why not implement it as discussed, using the attribute in tasks? or even in the root tag "selfdiagnose"?

Reply:
So within our services, we're not using the XML to configure Selfdiagnose, but are using the servlet. I don't know how many are using this library with the XML configuration. I could do both, if you want to.